server/controllers/games.js

```diff
@@ -0,0 +1,73 @@
+const Game = require('../models').Game;
+
+
+module.exports= {
+    create(req, res) {
+        return Game
+            .create({
+                game_number: req.body.game_number,
+                sr_change: req.body.sr_change,
```

Comment:
Though, I guess we'll need to discuss where we'll do these kinds of processing. Server side or client side?